doc/CHEATSHEET.md

```diff
@@ -0,0 +1,337 @@
+# Cheatsheet
+
+## Basics
+
+### Run a trace
+
+* In a test
+    * `Tasty.testCase "foo" $ C.testSucceeds foo`
+    * `Tasty.testCase "foo" $ C.testFails foo`
+* In the REPL
+    * `printCooked $ runMockChain foo`
+
+### Use a custom initial distribution of value
+
+```haskell
+initDist :: InitialDistribution
+initDist = initialDistribution [(i, [lovelaceValueOf 25_000_000]) | i <- knownWallets]
+```
+* In a test `Tasty.testCase "foo" $ testSucceedsFrom def initDist foo`
+* In the REPL `printCooked $ runMockChainFrom initDist foo`
+
+### Write a trace or endpoint
+
+```haskell
+foo :: MonadBlockChain m => m ()
+foo = do
+    transactionOrEndpoint1
+    transactionOrEndpoint2
+    transactionOrEndpoint3
+```
+
+### Get the current time
+
+```haskell
+foo :: MonadBlockChain m => m ()
+foo = do
+    (firstMsOfCurrentSlot, lastMsOfCurrentSlot) <- currentTime
```

Comment:
I strongly believe they should **NOT** need to be valid Haskell. This would, IMO, make those less "to the point". I would be in favor of adding a disclaimer and use `...` where applicable. I think every entry should be as small as possible. The future tutorial, however, should give examples that work.

Is there anything invalid here? Lack of `return ()`?
In this specific example, I think only information we want to convey is "use `currentTime`", "it is monadic" and "it returns the boundaries of the current slot". Adding a `return ()` would not help IMO.

Reply:
I have added a disclaimer and `...` in the code snippet to suggest it is not self contained. Tell me if this helps or not